modules/ROOT/pages/update-acm.adoc

```diff
@@ -6,18 +6,9 @@ endif::[]
 
 Keep your Anypoint API Community Manager instance up to date to take advantage of new and improved functionality in your community.
 
-For information about upgrades, see xref:notifications.adoc[Important Notifications].
+For information about upgrade notifications, see the documentation page on important notifications.
 
-After you upgrade using this process, you cannot undo the installation and return to a previous version. Follow this best-practice workflow:
-
-. Keep a partial copy sandbox or a full copy sandbox as a long-term staging environment.
-+
-The staging environment will have a partial or full copy of the data in the production environment.
-. Create a short-term developer sandbox.
-. Install the new package in the developer sandbox and test it.
-. Discard the developer sandbox.
-. Install the new package in the staging sandbox and test it.
-. Install the new package in your production environment.
+After you upgrade using this process, you cannot undo the installation and return to a previous version. Follow the best-practice workflow by creating safe sandbox environments and using them to test each upgrade before installing the new package in your production environment. For more information, see the documentation page on sandboxes.
```

Comment:
```suggestion
After you upgrade using this process, you cannot undo the installation and return to a previous version. Follow the best-practice workflow by creating safe sandbox environments and using them to test each upgrade before installing the new package in your production environment. For more information, see the documentation page on xref:sandboxes.adoc[sandboxes].
```

Reply:
OK, I've copied the links from the "See Also" section at the end into the main text.